examples/purchaseTesterTypescript/ios/PurchaseTester.xcodeproj/project.pbxproj

```diff
@@ -665,7 +665,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
```

Comment:
I wonder, any reason for this change?

Reply:
I was getting an issue somewhere mentioning a dependency required 13.4 but I don't remember if it was in `pod install` or running `yarn example ios`.

I think I am going to revert it and see if I see it again